index.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const parseArgs = (
+  argv = process.argv.slice(require.main ? 2 : 1),
+  options = {}
+) => {
+  if (typeof options !== 'object' || options === null) {
+    throw new Error('Whoops!')
```

Comment:
perhaps open a tracking ticket to add a clearer type checking message eventually 😆 (if you'd like to land this with Whoops!).